assets/survey_themes/fruity_twentythree/tables/no-tables-mobiles.scss

```
@@ -0,0 +1,134 @@
+
+    table.ls-answers > tbody > tr:hover {
+      background-color: transparent;
+    }
+
+    /* bs fix */
+    table.ls-answers td.visible-xs,
+    table.ls-answers th.visible-xs {
+      display: block !important
+    }
+
+    /* Don't display repeating header on mobile devices'*/
```

Stylelint (17.3.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)